### PR TITLE
Panel left shadow

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [x] Addresses an existing issue: #1555
- [x] Include a change request file if publishing 

### Before

image

### After 
image

### Same size on right

image

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Panel/Panel.scss [hunk 1]
```diff
@@ -92,6 +92,7 @@ $CommandBarHeight: 40px;
     @include right(auto);
     @include left(0);
     width: $Panel-width-xs;
+    @include drop-shadow(30px, 0px, 30px, -30px, .2);
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.scss]: Use the [`ms-box-shadow()` mixin](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/src/sass/mixins/_Directionality.Mixins.scss#L275) here, and on line 71, for RTL support.

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.scss]: I'll change both instances then

Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.scss]: My mistake, `drop-shadow()` is an RTL mixin already. :)

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.scss]: @mikewheaton not currently :)

```
@mixin drop-shadow($ms-x-offset: 0, $ms-y-offset: 0, $ms-blur: 5px, $ms-spread: 0, $ms-alpha: 0.4) {
  box-shadow: $ms-x-offset $ms-y-offset $ms-blur $ms-spread rgba(0, 0, 0, $ms-alpha);
}
```

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.scss]: Let's pull this in. Looks like core 7.0 will bring in the required mixing. We can update it then.

**Pull request merged**